proxygen/lib/utils/HTTPTime.cpp

```diff
@@ -25,10 +27,22 @@ folly::Optional<int64_t> parseHTTPDateTime(const std::string& s) {
   // Sun Nov 6 08:49:37 1994        ; ANSI C's asctime() format
   //    Assume GMT as per rfc2616 (see HTTP-date):
   //       - https://www.w3.org/Protocols/rfc2616/rfc2616-sec3.html
-  if (strptime(s.c_str(), "%a, %d %b %Y %H:%M:%S GMT", &tm) != nullptr ||
```

Comment:
There's no need to re-invent strptime, it's already implemented in `folly/portability/Time.h` (https://github.com/facebook/folly/blob/master/folly/portability/Time.h)